pkg/skuba/actions/node/join/join.go

```diff
@@ -71,29 +71,27 @@ func Join(joinConfiguration deployments.JoinConfiguration, target *deployments.T
 		criConfigure = "cri.configure"
 	}
 
-	statesToApply := []string{
-		"kernel.load-modules",
-		"kernel.configure-parameters",
-		"apparmor.start",
-		criConfigure,
-		"cri.start",
-		"kubelet.configure",
-		"kubelet.enable",
-		"kubeadm.join",
-		"skuba-update.start",
-	}
-
 	_, err = client.CoreV1().Nodes().Get(target.Nodename, metav1.GetOptions{})
 	if err == nil {
 		fmt.Printf("[join] failed to join the node with name %q since a node with the same name already exists in the cluster\n", target.Nodename)
 		return err
 	}
 
+	statesToApply := []string{"kubeadm.reset"}
+
 	if joinConfiguration.Role == deployments.MasterRole {
-		statesToApply = append([]string{"kubernetes.join.upload-secrets"}, statesToApply...)
+		statesToApply = append(statesToApply, "kubernetes.join.upload-secrets")
 	}
 
-	statesToApply = append([]string{"kubeadm.reset"}, statesToApply...)
+	statesToApply = append(statesToApply, "kernel.load-modules",
```

Comment:
Minor, can you please add a newline before `"kernel.load-modules"` as well?

Reply:
updated with changes from the comment